docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,262 @@
+# Bare Metal IPI (Installer Provisioned Infrastructure) Overview
+
+Current Status: **Experimental**
+
+This document discusses the installer support for an IPI (Installer Provisioned
+Infrastructure) install for bare metal hosts.  This includes platform support
+for the management of bare metal hosts, as well as some automation of DNS and
+load balancing to bring up the cluster.
+
+The upstream project that provides Kubernetes native management of bare metal
+hosts is [metal3.io](http://metal3.io).
+
+For UPI (User Provisioned Infrastructure) based instructions for bare metal
+deployments, see [install_upi.md](install_upi.md).
+
+## Prerequisites
+
+### Ironic
+
+Currently, the `baremetal` platform requires an existing Ironic environment.
+This will eventually be handled by `openshift-install`, with Ironic being
+deployed onto the bootstrap node. Until then, users of the `baremetal` platform
+should use the
+[openshift-metal3/dev-scripts](https://github.com/openshift-metal3/dev-scripts)
+repository to handle configuration of Ironic, and to launch the installer.
+
+The following PR contains the WIP changes for automating Ironic from
+`openshift-install`: https://github.com/openshift-metal3/kni-installer/pull/100
+
+### Network Requirements
+
+It is assumed that all hosts have at least 2 NICs, used for the following
+purposes:
+
+* **NIC #1 - External Network**
+  * This network is the main network used by the cluster, including API traffic
+    and application traffic.
+  * ***DHCP***
+    * External DHCP is assumed on this network.  It is **strongly** recommended
+      to set up DHCP reservations for each of the hosts in the cluster to
+      ensure that they retain stable IP addresses.
+    * A pool of dynamic addresses should also be available on this network, as
+      the provisioning host and temporary bootstrap VM will also need addresses
+      on this network.
+  * ***NTP***
+    * A time source must be accessible from this network.
+  * ***Reserved VIPs (Virtual IPs)*** - 3 IP addresses must be reserved on this
+    network for use by the cluster.  Specifically, these IPs will serve the
+    following purposes:
+    * API - This IP will be used to reach the cluster API.
+    * Ingress - This IP will be used by cluster ingress traffic
+    * DNS - This IP will be used internally by the cluster for automating
+      internal DNS requirements.
+  * ***External DNS*** - While the cluster automates the internal DNS
+    requirements, two external DNS records must be created in whatever DNS
+    server is appropriate for this environment.
+    * `api.<cluster-name>.<base-domain>` - pointing to the API VIP
+    * `*.apps.<cluster-name>.<base-domain>` - pointing to the Ingress VIP
+
+* **NIC #2 - Provisioning Network**
+  * A private, non-routed network, used for PXE based provisioning.
+  * DHCP is automated for this network.
```

Comment:
Should we mention this is presently hardcoded, but in the future we will let the user specify a range?

Reply:
Good idea